src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -258,6 +272,9 @@ const AccessibilityRequestDetailPage = () => {
         <UswdsLink href="#CreateAccessibilityRequestNote-NoteText">
           {t('requestDetails.notes.srOnlyAddNoteLink')}
         </UswdsLink>
+        <UswdsLink href="#AccessibilityRequestNotesTab-NotesList">
```

Comment:
I don't think these links work as intended. These links jump in scroll position, but don't shift focus. It should be shifting focus.

Reply:
Can you tell me more?  I don't know what they are supposed to behave like.  I did change it to jump to the h3 that Sai had me add, though I'm not sure if that's what you mean.